### ISSUE TITLE
Schedule: Event Layout - Command :: The Command field is malfunctioning

### ISSUE THREAD

**Issue by qaxlyka** [xibosignage/xibo#3557]
Please see the attached screen recording to check the actual issue encountered during testing. 

https://github.com/user-attachments/assets/9181faf8-ebb4-4f04-a938-2584f8647fbf

Steps:
1. Add Event
2. Change Event type as Command 
3. In the Command field, play around with it --> _the issue persist here_

**Comment by maurofmferrao** [comment]
This is a problem with `select2`, that doesn’t work well with Bootstrap modals and `tabIndex="-1"`

- https://select2.org/troubleshooting/common-problems
- https://github.com/xibosignageltd/xibo-private/issues/291

Unfortunately if the modal scrolls it will create a weird behaviour with the dropdown location, but I don’t think we can fix it manually. We need to wait for `select2` to release a patch to fix this issue.

**Comment by dasgarner** [comment]
> We need to wait for select2 to release a patch to fix this issue.

Does select2 have any plan to fix this? I don't see a linked bug report.

If they don't then we're essentially saying we won't fix this? In which case we should close it as Not Planned.

**Comment by maurofmferrao** [comment]
> > We need to wait for select2 to release a patch to fix this issue.
> 
> Does select2 have any plan to fix this? I don't see a linked bug report.
> 
> If they don't then we're essentially saying we won't fix this? In which case we should close it as Not Planned.

They know about the issue since they've added it to their documentation. And if they didn't create a ticket by now it's probably because they don't intend to fix it, should we close this then?

**Comment by dasgarner** [comment]
We either close it, or we fork and fix select2.

**Comment by maurofmferrao** [comment]
I don't think this is something easy to fix, since we would need to change the way select2 is calculating positioning with the zoom applied. Since Select2 also decided to ignore the issue, shall we mark is as "not planned" then?

**Comment by dasgarner** [comment]
Closing as not planned - fixing would involve moving away from select2, or forking and fixing, both of which are significant issues.
